http_handler_file.cc

```diff
@@ -0,0 +1,75 @@
+#include <fstream>
+#include "http_handler_file.h"
+#include "http_request.h"
+#include "http_response.h"
+
+
+namespace http {
+
+
+// Structure for mapping extensions to content types
+struct mapping {
```

Comment:
I like how you used a struct to add mappings instead of using a standard library data structure